torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -1223,8 +1223,8 @@ std::vector<at::Tensor> FusionExecutor::runFusion(
   if (isDebugDumpEnabled(DebugDumpOption::KernelArgs)) {
     std::cout << "Arguments for kernel" << fusion_id_ << ":" << std::endl
               << "Inputs:" << std::endl;
-    for (auto i : c10::irange(args.size())) {
-      args[i]->print();
+    for (auto i : c10::irange(num_inputs)) {
```

Comment:
Haha, thx for this. you beat me to it. https://github.com/pytorch/pytorch/issues/86529#issuecomment-1299115715